src/services/get-songs.ts

```diff
@@ -210,7 +210,7 @@ export default class {
       case 'playlist': {
         const uri = parsed as spotifyURI.Playlist;
 
-        let [{body: playlistResponse}, {body: tracksResponse}] = await Promise.all([this.spotify.getPlaylist(uri.id), this.spotify.getPlaylistTracks(uri.id, {limit: 50})]);
+        let [{body: playlistResponse}, {body: tracksResponse}] = await Promise.all([this.spotify.getPlaylist(uri.id), this.spotify.getPlaylistTracks(uri.id, {limit})]);
```

Comment:
We're already fetching all tracks in the playlist through pagination, this needs to to be reverted.

Reply:
> I was thinking maybe there still should be an upperbound for this setting. I came across this error when setting the limit to 150

(Caused by this, maximum supported value on Spotify's side is 50 per call.)